Simple-Projects/Simple_Projects/Basic/List/List.cpp

```diff
@@ -28,28 +27,23 @@ int List::max_size() const {
 }
 
 void List::erase(int index) {
-    if ( current > index && index > 0 ) {
-        current -= 1;
-
-        for (int i = 0; i < current; i++ ) {
-        	array[i] = array[i+1];
-        }
+    if ( current > index && index >= 0 ) {
+		std::copy(array + index + 1, array + current, array);
+        --current;
     }
 }
 
 void List::insert(int value, int index) {
     if ( current >= index && index >= 0 ) {
-        push_back(value);
+        push_back(0);
```

Comment:
The value pushed back is unused, so why pass it? Might as well have something hard-coded